armv7a9-zynq7000/serial.c

```diff
@@ -21,6 +21,8 @@
 #include "../low.h"
 
 
+#define ARRAYSIZE(a)  (sizeof(a) / sizeof(a[0]))
+
```

Comment:
We have guidlines agains using such macro (it's common name is NELEMS()). If you use sizeof, please use it explicitly.